### ISSUE TITLE
Oak crashes sometimes

### ISSUE THREAD

**Issue by hsjoberg** [hsjoberg/fork-explorer#11]
```
run-server.sh[208373]: error: Uncaught (in promise) Http: connection closed before message completed
run-server.sh[208373]:           for await (const requestEvent of httpConn) {
run-server.sh[208373]:                            ^
run-server.sh[208373]:     at unwrapOpResult (deno:core/core.js:99:13)
run-server.sh[208373]:     at async HttpConn.nextRequest (deno:runtime/js/40_http.js:33:23)
run-server.sh[208373]:     at async Object.next (deno:runtime/js/40_http.js:83:26)
run-server.sh[208373]:     at async serve (https://deno.land/x/oak@v7.3.0/http_server_native.ts:161:28)
```

**Comment by hsjoberg** [comment]
Seems related to https://github.com/denoland/deno/pull/10384

**Comment by hsjoberg** [comment]
The issue is addressed by temporarily compiling deno v1.9.2 cherry-picked with https://github.com/denoland/deno/pull/10384 on the production server for https://taproot.watch.

**Comment by hsjoberg** [comment]
Issue fixed upstream in deno v1.13.3.
